tests/conftest.py

```diff
@@ -1069,6 +1073,8 @@ def factory(
                 dc_deployment=deployment_config,
                 sa_name=sa_name,
                 replica_count=replica_count,
+                pod_name=pod_name,
```

Comment:
as discussed please check if we really need it?

Reply:
pod_name argument was already part of create_pod method but i noticed that our pod_factory method doesnt have that option as argument which makes create_pod method argument useless. so I thought ill add it if anyone wants to create a pod with custom name. Having said that, we can make this test work without custom custom pod name too :D